src/components/index.js

```diff
@@ -3,4 +3,7 @@ import { ButtonCta } from "./ButtonCta";
 import { Header } from "./Header";
 import { PageContainer } from "./PageContainer";
 import { NotesdModal } from "./NotesModal";
-export {AppNavigation,ButtonCta,Header,PageContainer,NotesdModal}
\ No newline at end of file
+import { EmptyState } from './EmptyState';
+import { PageHeader } from './PageHeader';
+import { NotesContainer, NoteCard } from './NoteCard'
+export {AppNavigation,ButtonCta,Header,PageContainer,NotesdModal,EmptyState,PageHeader,NotesContainer,NoteCard}
```

Comment:
You can directly export, without importing, using barrel export.

Reply:
thanks saran i did not know it earlier